slices/tzdata.yaml

```diff
@@ -32,20 +32,227 @@ slices:
+      /usr/share/zoneinfo/America/Martinique:
+      /usr/share/zoneinfo/America/Matamoros:
+      /usr/share/zoneinfo/America/Mazatlan:
+      /usr/share/zoneinfo/America/Menominee:
+      /usr/share/zoneinfo/America/Merida:
+      /usr/share/zoneinfo/America/Metlakatla:
+      /usr/share/zoneinfo/America/Mexico_City:
+      /usr/share/zoneinfo/America/Miquelon:
+      /usr/share/zoneinfo/America/Moncton:
+      /usr/share/zoneinfo/America/Monterrey:
+      /usr/share/zoneinfo/America/Montevideo:
+      /usr/share/zoneinfo/America/Montreal:
+      /usr/share/zoneinfo/America/Montserrat:
+      /usr/share/zoneinfo/America/Nassau:
+      /usr/share/zoneinfo/America/New_York:
+      /usr/share/zoneinfo/America/Nipigon:
+      /usr/share/zoneinfo/America/Nome:
+      /usr/share/zoneinfo/America/Noronha:
+      /usr/share/zoneinfo/America/North_Dakota/**:
+      /usr/share/zoneinfo/America/Nuuk:
+      /usr/share/zoneinfo/America/Ojinaga:
+      /usr/share/zoneinfo/America/Panama:
+      /usr/share/zoneinfo/America/Pangnirtung:
+      /usr/share/zoneinfo/America/Paramaribo:
+      /usr/share/zoneinfo/America/Phoenix:
+      /usr/share/zoneinfo/America/Port-au-Prince:
+      /usr/share/zoneinfo/America/Port_of_Spain:
+      /usr/share/zoneinfo/America/Porto_Acre:
+      /usr/share/zoneinfo/America/Porto_Velho:
+      /usr/share/zoneinfo/America/Puerto_Rico:
+      /usr/share/zoneinfo/America/Punta_Arenas:
+      /usr/share/zoneinfo/America/Rainy_River:
+      /usr/share/zoneinfo/America/Rankin_Inlet:
+      /usr/share/zoneinfo/America/Recife:
+      /usr/share/zoneinfo/America/Regina:
+      /usr/share/zoneinfo/America/Resolute:
+      /usr/share/zoneinfo/America/Rio_Branco:
+      /usr/share/zoneinfo/America/Santa_Isabel:
+      /usr/share/zoneinfo/America/Santarem:
+      /usr/share/zoneinfo/America/Santiago:
+      /usr/share/zoneinfo/America/Santo_Domingo:
+      /usr/share/zoneinfo/America/Sao_Paulo:
+      /usr/share/zoneinfo/America/Scoresbysund:
+      /usr/share/zoneinfo/America/Shiprock:
+      /usr/share/zoneinfo/America/Sitka:
+      /usr/share/zoneinfo/America/St_Barthelemy:
+      /usr/share/zoneinfo/America/St_Johns:
+      /usr/share/zoneinfo/America/St_Kitts:
+      /usr/share/zoneinfo/America/St_Lucia:
+      /usr/share/zoneinfo/America/St_Thomas:
+      /usr/share/zoneinfo/America/St_Vincent:
+      /usr/share/zoneinfo/America/Swift_Current:
+      /usr/share/zoneinfo/America/Tegucigalpa:
+      /usr/share/zoneinfo/America/Thule:
+      /usr/share/zoneinfo/America/Thunder_Bay:
+      /usr/share/zoneinfo/America/Tijuana:
+      /usr/share/zoneinfo/America/Toronto:
+      /usr/share/zoneinfo/America/Tortola:
+      /usr/share/zoneinfo/America/Vancouver:
+      /usr/share/zoneinfo/America/Virgin:
+      /usr/share/zoneinfo/America/Whitehorse:
+      /usr/share/zoneinfo/America/Winnipeg:
+      /usr/share/zoneinfo/America/Yakutat:
+      /usr/share/zoneinfo/America/Yellowknife:
       /usr/share/zoneinfo/posixrules:
```

Comment:
I think we should move this to etc. I know this is NY, but posixrules is in a location based slice.

Reply:
Discussed offline to keep it here since this already existed in this slice and similar structure is also present in other releases.